_data/sidebar.yml

```diff
@@ -1713,39 +1746,47 @@
   isSectionHeader: 0
   isCatHeader: 1
   sectionTitle:
-  subgroup: 1
+  subgroup: 2
 
 - sbSecId: 4
   title: 'Basic IMA'
   link: /examples/video/outstream/basic-ima.html
   isHeader: 0
   isSectionHeader: 0
   sectionTitle:
-  subgroup: 1
+  subgroup: 2
 
 - sbSecId: 4
   title: 'Google Ad Manager'
   link: /examples/video/outstream/pb-ve-outstream-dfp.html
   isHeader: 0
   isSectionHeader: 0
   sectionTitle:
-  subgroup: 1
+  subgroup: 2
+
+- sbSecId: 4
+  title: 'Radiant'
```

Comment:
@muuki88 I cherry-picked your commits. For some reason this `Radiant` section appears to be new, however the file in master has it.

Reply:
I don't think this should be an issue